modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc

```diff
@@ -14,7 +14,8 @@ The following environment variable is mandatory:
 - `BONITA_API_URL`: The URL for the Bonita API.
 
 The following environment variables are optional:
-
+- `BONITA_HEALTHCHECK_USER`: Defines the username to request the Bonita healthcheck API (default: `monitoring`).
```

Comment:
The layout is broken :scream:  Add an empty line before the 1st element of the list

Reply:
good catch 👍